tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/modules/LiferayPortletModuleCustomPart.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.ui.modules;
+
+import org.eclipse.swt.widgets.Shell;
+
+/**
+ * @author Simon Jiang
+ */
+public class LiferayPortletModuleCustomPart extends LiferayPropertyModuleCustomPart
```

Comment:
Yeah for now for "properties" we can just have a normal sapphire table like
the table editors in sapphire gallery.  Since it is simply key/value
pairs.  Just some of our table editors in portlet.xml editors.

On Wed, Dec 23, 2015 at 8:20 PM, simon.jiang <notifications@github.com>
wrote:

> In
> tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/modules/LiferayPortletModuleCustomPart.java
> <https://github.com/gamerson/liferay-ide/pull/1262#discussion_r48392362>:
>
> > + *
> > + * This library is distributed in the hope that it will be useful, but WITHOUT
> > + * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
> > + * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
> > + * details.
> > + *
> > + *******************************************************************************/
> > +
> > +package com.liferay.ide.project.ui.modules;
> > +
> > +import org.eclipse.swt.widgets.Shell;
> > +
> > +/**
> > + * @author Simon Jiang
> > + */
> > +public class LiferayPortletModuleCustomPart extends LiferayPropertyModuleCustomPart
>
> greg, you mean we don't need to provide the swt add, edit and remove
> button style for user add, edit or remove properties, just directly use
> saffire default table layout?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/1262/files#r48392362>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com